src/SideAppbar/SideAppbar.js

```diff
@@ -0,0 +1,130 @@
+      name: "VIRTO.ID"
+    }
+  ];
+
+  // handle button click
+  const handleClick = name => {
+    onButtonClick(name);
+  };
+
+  return (
+    <>
+      {showLogo ? (
+        <>
+          {logoLinkUrl ? (
+            <Link href={logoLinkUrl} underline="none">
+              <AppLogo />
+            </Link>
+          ) : (
+            <AppLogo />
+          )}
+        </>
+      ) : null}
+      <Box display="flex" flexWrap="wrap" justifyContent="space-between">
+        {appList.map(app => {
+          return (
+            <Paper
+              key={app.name}
+              data-testid={app.name}
+              sx={theme => ({
+                "&:hover": {
+                  boxShadow: theme.shadows[1]
+                },
+                boxShadow: "none",
+                cursor: "pointer",
+                mb: 2,
+                width: 120
+              })}
+              onClick={() => handleClick(app.name)}
+            >
+              <Box sx={{ px: 2, py: 1.5, textAlign: "center" }}>
```

Comment:
Something not quite right happening here when the text is slightly too long:

![image](https://user-images.githubusercontent.com/27866636/221561277-37c9b29e-80bf-41ec-847f-2d09c661b737.png)